odoo/README.md

```diff
@@ -0,0 +1,111 @@
+# Supported tags and respective `Dockerfile` links
+
+- [`8.0`, `8`, `latest` (*8.0/Dockerfile*)](https://github.com/odoo/docker/blob/0c90527ef908add1f125406a81519f9626d982d3/8.0/Dockerfile)
+
+For more information about this image and its history, please see the [relevant
+manifest file
+(`library/odoo`)](https://github.com/docker-library/official-images/blob/master/library/odoo)
+in the [`docker-library/official-images` GitHub
+repo](https://github.com/docker-library/official-images).
+
+# What is Odoo?
+
+Odoo, formerly known as OpenERP, is a suite of open-source business apps 
+written in Python and released under the AGPL license. This suite of 
+applications covers all business needs, from Website/Ecommerce down to 
+manufacturing, inventory and accounting, all seamlessly integrated. It is the 
+first time ever a software editor managed to reach such a functional coverage. 
+Odoo is the most installed business software in the world. Odoo is used by 
+2.000.000 users worldwide ranging from very small companies (1 user) to very 
+large ones (300 000 users). 
+
+> [www.odoo.com](https://www.odoo.com)
+
+![logo](https://raw.githubusercontent.com/docker-library/docs/master/odoo/logo.png)
+
+# How to use this image
+
+This image requires a running PostgreSQL server. 
+
+## Start a PostgreSQL server
+
+	docker run -d -e POSTGRES_USER=odoo -e POSTGRES_PASSWORD=odoo --name db postgres
+
+## Start an Odoo instance
+
+	docker run -p 127.0.0.1:8069:8069 --name odoo --link db:db -t odoo
+
+The alias of the container running Postgres must be db for Odoo to be able
+to connect to the Postgres server.
+
+## Stop and restart an Odoo instance
+	
+	docker stop odoo
+	docker start -a odoo
+
+## Stop and restart a PostgreSQL server
+When a PostgreSQL server is restarted, the Odoo instances 
+linked to that server must be restarted as well because the server address has 
+changed and the link is thus broken. 
+
+Restarting a PostgreSQL server does not affect the created databases.
+
+## Run Odoo with a custom configuration
+
+The default configuration file for the server (located at `/etc/odoo/openerp-server.conf`)
+can be overriden at startup using volumes. Suppose you have a custom configuration
+at `/path/to/config/openerp-server.conf`, then
+
+	docker run -v /path/to/config:/etc/odoo -p 127.0.0.1:8069:8069 --name odoo --link db:db -t odoo
+
+## Run multiple Odoo instances
+	
+	docker run -p 127.0.0.1:8070:8069 --name odoo2 --link db:db -t odoo
+	docker run -p 127.0.0.1:8071:8069 --name odoo3 --link db:db -t odoo
```

Comment:
It will work even if it doesn't really make sense... The classic use case would be to create a database for each odoo instance.